src/appleseed.studio/mainwindow/project/entityeditor.cpp

```diff
@@ -516,6 +516,7 @@ unique_ptr<IInputWidgetProxy> EntityEditor::create_colormap_input_widgets(const
     ColorMapInputWidget* input_widget = new ColorMapInputWidget(m_parent);
     input_widget->set_validator(new QDoubleValidatorWithDefault(validator_min, validator_max, 16, default_value));
     input_widget->set_default_value(default_value);
+    input_widget->set_slider_range(slider_min, slider_max);
```

Comment:
While that looked an oversight, the reason why the range stays [0, 1] for color map inputs is that it's the range of values that a texture can provide. This is why often a color map input is accompanied by a scalar multiplier input with the correct range (in the case of exposures, [-64, 64]).

However the `slider_min` and `slider_max` variables are clearly unused and can be removed.